README.md

```diff
@@ -64,6 +64,25 @@ To use this template:
 
 > The setup script removes the `## Explainer` and `## Setup` from this README.md.
 
+### Setup Options
+
+The setup script requires four options to fill out repository details.
+It will interactively prompt for any that are not provided as a string CLI flag:
+
+1. `repository`: The kebab-case name of the repository (e.g. `template-typescript-node-package`)
+2. `title`: Title Case title for the repository to be used in documentation (e.g. `Template TypeScript Node Package`)
+3. `owner`: GitHub organization or user the repository is underneath (e.g. `JoshuaKGoldberg`)
+4. `description`: Sentence case description of the repository (e.g. `A quickstart-friendly TypeScript package with lots of great repository tooling. ✨`)
+
+Additionally, a `--skip-api` boolean CLI flag may be specified to prevent the setup script from calling to GitHub APIs for repository hydration.
+Doing so limits setup changes to local files in Git: which can be useful to preview what running setup does.
```

Comment:
I think I understand what this means but maybe I don't. Is skip-api effectively skip-git-push? Or is there more to it? Will I regret it if I do skip? Can I recover if I do?

Reply:
> effectively skip-git-push

It won't `git push` either way. The setup script normally posts to the GitHub API to hydrate the repo on github.com. This prevents it.

> Will I regret it if I do skip? Can I recover if I do?

You can always `git add -A; git reset --hard HEAD~ or similar.

I'll add some notes!